wallet/tickets.go

```diff
@@ -129,11 +131,59 @@ func (w *Wallet) TicketHashesForVotingAddress(votingAddr dcrutil.Address) ([]cha
 	return ticketHashes, err
 }
 
+// UpdateStakePoolInvalidTicket properly updates a previously marked Invalid pool ticket,
+// it then creates a new entry in the validly tracked pool ticket db.
+func (w *Wallet) UpdateStakePoolInvalidTicket(stakemgrNs walletdb.ReadWriteBucket, addrmgrNs walletdb.ReadBucket,
```

Comment:
Make this unexported or find a way to remove the namespace parameters.  Exported wallet APIs should be usable by other packages and this one is not because the db is not accessible.